src/state/state.rs

```diff
@@ -229,7 +226,6 @@ impl<B: DB> State<B> {
 
     /// Add balance by incr for an account.
     pub fn add_balance(&mut self, address: &Address, incr: U256) -> Result<(), Error> {
-        debug!("state.add_balance a={:?} incr={:?}", address, incr);
```

Comment:
Why delete those debug info? I think it must be very useful while detecting problems in the future.